README.md

```diff
@@ -21,6 +21,13 @@ For example, if you see `types "sigs.k8s.io/cluster-api/vendor/k8s.io/apimachine
 
 You can use Google Cloud or AWS ECR to host repositories.
 
+1. Install [jq](https://stedolan.github.io/jq/download/) (`brew install jq` on Mac)
+2. Install `gettext` package (`brew install gettext && brew link --force gettext` on Mac)
+3. Install `minikube` (`brew install minikube`)
+4. Configure `minikube` 
```

Comment:
Any particular reason to explicitly number these instead of using `1.` for all and letting the auto-numbering do it's thing?

Reply:
Not really, I just saw that in other places it's also like this, so followed the convention